scormxblock/static/js/src/scormxblock.js

```diff
@@ -127,20 +127,43 @@ function ScormXBlock_${block_id}(runtime, element) {
       }
       return false;
     }
+
     function disableNextModuleArrow() {
-      var next_link = document.getElementsByClassName("next")[0];
-      var name = "complete-scorm future";
-      arr = next_link.className.split(" ");
-      if (arr.indexOf(name) == -1) {
-        next_link.className += " " + name;
+      if (isNewUI()) {
+        disableNextModuleArrowForNewUI();
+      }
+      else {
+        disableNextModuleArrowForOldUI();
       }
+    }
+
+    function isNewUI() {
+      return $( "body.new-theme" ).length;
+    }
+
+    function disableNextModuleArrowForOldUI() {
+      var next_module_link = $( ".controls .next" );
+      next_module_link.addClass( "complete-scorm future" );
+      next_module_link.attr("href", "#");
```

Comment:
In future, we need to enable the next button on runtime. Emptying the href would result in complications. How about we set the background color of the button to grey and set `pointer-events: none` in css. The `disabled` css class in old ui would do that.

Reply:
We need pointer events since we want to display popup on click. Also in future we can save this in another attribute to retrieve it later and their is NO COMPLICATIONS in it. The disabled css class in old ui doesn't suit for this use case.